src/main/java/com/spanish_inquisition/battleship/network/SocketClient.java

```diff
@@ -0,0 +1,34 @@
+package com.spanish_inquisition.battleship.network;
+
+import com.spanish_inquisition.battleship.common.AppLogger;
+
+import java.io.*;
+import java.net.Socket;
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author Michal_Partacz
+ */
+public class SocketClient {
+    private final static Logger logger = Logger.getLogger(AppLogger.class.getName());
```

Comment:
See comment above. In log there is an information about class which is basically where problem occured.